content/guides/content-management/permission-editor.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Permission editor"
+metaTitle: "sensenet admin-ui - Permission editor"
+metaDescription: "sensenet admin-ui - Permission editor"
+---
+# Permission editor
+
+sensenet offers a sophisticated permission system, giving you the possibility to apply permission rules according to the logic you prefer. The permission editor is available on the admin ui.
+
+There are a lot of built-in permissions but there are some reserved ones that you are free to use in case you have special needs (custom permissions). You can also group these permissions for the permission editor. The related settings file is available under Setup menu / Permission.settings.
```

Comment:
This part is a bit _in medias res_.  A little introduction or references to the other permission related concept docs are missing here.